### PR TITLE
resize canvas upon window resize

### PR DESCRIPTION
**Pull request by cvan**

### CODE REVIEW

File: renderer.js [hunk 1]
```diff
@@ -18,6 +18,7 @@ function initCanvas () {
   canvas.width = window.innerWidth * 0.5;
   canvas.addEventListener('click', enterFullscreen);
   document.addEventListener('mozfullscreenchange', exitFullscreen);
+  window.addEventListener('resize', handleResize);  // TODO: Consider a debounce.
```
Comment (nickdesaulniers) [inline renderer.js]: then require debounce like main.js does: https://github.com/nickdesaulniers/joshVR/blob/master/main.js#L6
it's already a dependency: https://github.com/nickdesaulniers/joshVR/blob/master/package.json#L7

**Comment by nickdesaulniers** [comment]
ugh, idk what I just did, other than mess up history

**Comment by nickdesaulniers** [comment]
merged

The trick was to:
1. click command line instructions
2. click Patch
3. copy url
4. git apply <url>
5. git merge --amend
6. git push

I think...

**Comment by cvan** [comment]
heh, yeah let's just click the big green merge button from now on. merge commits are okay.

**Pull request merged**